tests/subsys/net/lib/lwm2m_client_utils/testcase.yaml

```diff
@@ -0,0 +1,4 @@
+tests:
+  subsys.net.lib.lwm2m_client_utils.unittest:
+    platform_allow: native_posix
```

Comment:
Please add `integration_platforms` option here, to make it possible to run those tests in CI
```yaml
integration_platforms:
  - native_posix
```

Reply:
added